### PR TITLE
feat: #84 rp adapter: parse vp from CHAPI response

### LINKED ISSUE #84: rp adapter: parse verifiable presentation from CHAPI response
part of #60

### PR DESCRIPTION
**Pull request by llorllale**
closes #84

Signed-off-by: George Aristy <george.aristy@securekey.com>

### CODE REVIEW

File: pkg/restapi/rp/operation/credentials.go [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+	"github.com/hyperledger/aries-framework-go/pkg/doc/verifiable"
+	"github.com/pkg/errors"
+
+	"github.com/trustbloc/edge-adapter/pkg/vc"
+)
+
+var errMalformedCredential = errors.New("malformed credential")
+
+//nolint:unparam
+func getCustomCredentials(vpBytes []byte) (*vc.DIDDocumentCredential, *vc.UserConsentCredential, error) {
```
Comment (llorllale) [inline pkg/restapi/rp/operation/credentials.go:23]: this will be plugged into `presentationResponseHandler`

**Comment by llorllale** [comment]
I'm not sure how to increase coverage on this one.

File: pkg/vc/models.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package vc
```
Comment (rolsonquadras) [inline pkg/vc/models.go:7]: Can we place these inside `pkg/vc/rp` ? We gonna have issuer credentials as well and there are no common elements between issuer and rp.

Comment (llorllale) [reply pkg/vc/models.go:7]: `UserConsentCredential` is common to both

Comment (llorllale) [reply pkg/vc/models.go:7]: @rolsonquadras moved `DIDDocumentCredential` to `pkg/vc/rp`

**Pull request merged**